site/src/docs/components/highlight/tabs.mdx

```diff
@@ -0,0 +1,30 @@
+---
+slug: '/components/highlight/tabs'
+title: 'Highlight'
+---
+
+import { StatusLabel } from 'hds-react';
+
+import LeadParagraph from '../../../components/LeadParagraph';
+import Layout from '../../../components/layout';
+import PageTabs from '../../../components/PageTabs';
+
+# Highlight
+
+<StatusLabel type="alert" variant="rounded">Beta</StatusLabel>
+
+<LeadParagraph>
+    Highlight component allows lifting an excerpt from a longer text for emphasis. Highlight supports two types: highlight (default) and quotation. The Quote element has a reference to the quote source. The Highlight element is an excerpt from the text without the source information.
+</LeadParagraph>
```

Comment:
To remove console warning about <p> inside a <p> tag, remove the new lines from the start and end of this paragraph. This kind of formatting is probably done by prettier but it causes said warnings.